### PR TITLE
Adding absolute magnitude functionality

### LINKED ISSUE #9: Add option for user to input absolute magnitude instead of apparent magnitude
I may also have to modify a bunch of the print statements to specify that they are referring to apparent magnitude

### PR DESCRIPTION
**Pull request by adamtrapp**
Users may now input mag as absolute AB magnitudes if they set the flag appOrAbs = 1. Also, changing setup file to indicate that scipy is now a required package

### CODE REVIEW

**Review by JohnTrapp — COMMENTED** [review]
"Apparent magnitude or absolute magnitude" isn't a yes/no question, so `appOrAbs = 1` is confusing as it's not clear what "1" means.

Instead, using `useAbsoluteMag = [0|1]` or a string value `magnitude = [absolute|apparent]` would be clearer and provide a better UX.

File: galcv/__init__.py [hunk 1]
```diff
@@ -371,3 +378,70 @@ def log10Eps(area, Psi, gamma, b):
         The parameters of the varepsiloncv fit
     '''
     return Psi * area**gamma + b
+
+
+def absToApp(Mabs='ER', z='ER'):
```
Comment (JohnTrapp) [inline galcv/__init__.py:387]: These new methods would be an excellent time to start breaking this file apart into more cohesive and distinct files/modules.

Comment (adamtrapp) [reply galcv/__init__.py:387]: That's true. That will probably take a little work to implement and make sure I'm doing it the correct way. Lets leave that to a future pull request. I'll add it as an issue.

**Comment by adamtrapp** [comment]
I changed the appOrAbs keyword to be a string

**Pull request merged**